CHANGELOG.md

```diff
@@ -66,6 +66,25 @@ Thanks to [#5342](https://github.com/decidim/decidim/pull/5342), Decidim now sup
 
 **Fixed**:
 
+- **decidim-admin**: [FIX] consultation queries don't result in exception [\#5561](https://github.com/decidim/decidim/pull/5561)
+- **decidim-conferences**: Fix: Cluttered conference sessions in confirmation mail.[\#5524](https://github.com/decidim/decidim/pull/5524)
```

Comment:
Watch out with these changes! They were imported when merging from `master`, but shouldn't be here 😄